core-bundle/contao/templates/twig/content_element/swiper.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+{% use "@Contao/component/_stylesheet.html.twig" %}
+
+{% block content %}
+    <div class="swiper" data-delay="{{ delay }}" data-speed="{{ speed }}" data-offset="{{ offset }}" data-loop="{{ loop }}">
+        <div class="swiper-wrapper">
+            {% for element in nested_fragments %}
+                <div class="swiper-slide">
+                    {{ content_element(element) }}
+                </div>
+            {% endfor %}
+        </div>
+        <div class="swiper-button-next"></div>
+        <div class="swiper-button-prev"></div>
```

Comment:
```suggestion
        <button type="button" class="swiper-button-next"></button>
        <button type="button" class="swiper-button-prev"></button>
```
Using buttons made it possible to use the keyboard (<kbd>space</kbd> or <kbd>enter</kbd>) to activate the arrows in my test